recipes/impyla/meta.yaml

```diff
@@ -0,0 +1,44 @@
+package:
+  name: impyla
+  version: "0.13.6"
+
+source:
+  fn: impyla-0.13.6.tar.gz
+  url: https://pypi.python.org/packages/source/i/impyla/impyla-0.13.6.tar.gz
+  md5: 56b5889595a3c1bfc7700615935d264f
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - six
+    - bitarray
+    - thrift  # [py2k]
+    - thriftpy >=0.3.5
```

Comment:
Is this a required or optional dependency? If it is required, we will need to skip Windows here too as shown on `thriftpy`. If it is optional, we can just skip this dependency on Windows by adding `# [not win]` here.